app/src/main/python/gogdl/saves.py

```
@@ -0,0 +1,365 @@
+                files.append(abs_path)
+        return files
+
+    @staticmethod
+    def get_relative_path(root: str, path: str) -> str:
+        if not root.endswith("/") and not root.endswith("\\"):
+            root = root + os.sep
+        return path.replace(root, "")
+
+    def sync(self, arguments, unknown_args):
+        try:
+            prefered_action = getattr(arguments, 'prefered_action', None)
```

Ruff (0.14.8)

Local variable `prefered_action` is assigned to but never used

Remove assignment to unused variable `prefered_action`

(F841)